scripts/scil_compute_kurtosis_metrics.py

```diff
@@ -0,0 +1,322 @@
+biases due to fiber dispersion, fiber crossings and other mesoscopic properties
+of the underlying tissue, MSDKI does a powder-average of DWI for all
+directions, thus removing the orientational dependencies and creating an
+alternative mean kurtosis map.
+
+DKI is also known to be vulnerable to artefacted voxels induced by the
+low radial diffusivities of aligned white matter (CC, CST voxels). Since it is
+very hard to capture non-Gaussian information due to the low decays in radial
+direction, its kurtosis estimates have very low robustness.
+Noisy kurtosis estimates tend to be negative and its absolute values can have
+order of magnitudes higher than the typical kurtosis values. Consequently,
+these negative kurtosis values will heavily propagate to the mean and radial
+kurtosis metrics. This is well-reported in [Rafael Henriques MSc thesis 2012,
+chapter 3]. Two ways to overcome this issue: i) compute the kurtosis values
+from powder-averaged MSDKI, and ii) perform 3D Gaussian smoothing. On
+powder-averaged signal decays, you don't have this low diffusivity issue and
+your kurtosis estimates have much higher precision (additionally they are
+independent to the fODF).
+
+By default, will output all available metrics, using default names. Specific
+names can be specified using the metrics flags that are listed in the "Metrics
+files flags" section. If --not_all is set, only the metrics specified
+explicitly by the flags will be output.
+
+This script directly comes from the DIPY example gallery and references
+therein.
+[1] examples_built/reconst_dki/#example-reconst-dki
+[2] examples_built/reconst_msdki/#example-reconst-msdki
+"""
+
+from __future__ import division, print_function
+
+from builtins import range
+import argparse
+import logging
+
+import nibabel as nib
+import numpy as np
+
+import dipy.reconst.dki as dki
+import dipy.reconst.msdki as msdki
+
+from dipy.io.gradients import read_bvals_bvecs
+from dipy.core.gradients import gradient_table
+
+from scipy.ndimage.filters import gaussian_filter
+
+# Aliased to avoid clashes with images called mode.
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_outputs_exist, add_force_b0_arg)
+from scilpy.utils.bvec_bval_tools import (normalize_bvecs, is_normalized_bvecs,
+                                          check_b0_threshold)
+
+logger = logging.getLogger("Compute_DKI_Metrics")
+logger.setLevel(logging.INFO)
+
+
+def _guess_bvals_centroids(bvals, threshold):
+    if not len(bvals):
+        raise ValueError('Empty b-values.')
+
+    bval_centroids = [bvals[0]]
+
+    for bval in bvals[1:]:
+        diffs = np.abs(np.asarray(bval_centroids) - bval)
+        # Found no bval in bval centroids close enough to the current one.
+        if not len(np.where(diffs < threshold)[0]):
+            bval_centroids.append(bval)
+
+    return np.array(bval_centroids)
+
+
+def identify_shells(bvals, threshold=40.0):
```

Comment:
Can you add this function in scilpy/utils/bvec_bval_tools.py and can you add the documentation to the function please